### PR TITLE
171/implement a consistent layout for the homepage and resultpage for the webversion

### PR DESCRIPTION
**Pull request by MaiThiD**

### CODE REVIEW

File: app/src/views/Home.vue [hunk 1]
```diff
@@ -18,8 +18,8 @@
 
     <v-container>
       <v-form>
-        <v-row justify="center">
-          <v-col cols="12" md="8">
+        <v-row id="searchbox" justify="center" lg="2">
+          <!-- v-col cols="12" md="8"-->
```
Comment (FiedlerJan) [inline app/src/views/Home.vue:22]: Please remove commented code.

File: app/src/views/Home.vue [hunk 2]
```diff
@@ -185,4 +185,17 @@ img {
...
+
+@media (min-width: 800px)
+{
+.container {
+  max-width:85%
+}
+#searchbox {
+  padding-left:12px;
+  padding-right:12px;
+}
```
Comment (FiedlerJan) [inline app/src/views/Home.vue:197]: Please use correct indents.

File: app/src/views/ResultPage.vue [hunk 3]
```diff
@@ -477,4 +476,57 @@
...
+  #mapcard {
+  width:83%; 
+  margin-top: 5%; 
+  margin-right:15%;
+}
+  #postopenright {
+  width:83%; 
+  margin-top: 5%; 
+  margin-right:15%; 
+}
```
Comment (FiedlerJan) [inline app/src/views/ResultPage.vue:527]: Please use correct indents.

**Review by FiedlerJan — CHANGES_REQUESTED** [review]
Looks good, but still has some small things that should be changed.

**Review by FiedlerJan — APPROVED** [review]

**Review by LeDrieHda — APPROVED** [review]
From the viewpoint of functional architect:
UI on home page and result page is consistent in that they have the same margin left and right.
Result page has implemented the proposed and discussed changes and its design is now overall more appealing to the user.

**Review by DianaPankratz — APPROVED** [review]

**Pull request merged**